### PR TITLE
Archive functionality

### PR DESCRIPTION
**Pull request by sharathnair9999**
Features:

Count of each section on the tab. (Total notes, Archived counts implemented as of now)
Add and restoring from and to Archives

### CODE REVIEW

**Review by ArjunGTX — APPROVED** [review]
Everything looks good. Explainatory error messages and fallback texts for empty list

**Review by KanekarSnehal — APPROVED** [review]
Code is well structured.
I liked how you displayed current time also on each note.
Overall code looks good no issues found you can merge it.

File: src/pages/Archives/ArchivesPage.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
 const ArchivesPage = () => {
+  const { noteState } = useNotes();
+  const { archivedNotes } = noteState;
   return (
-    <div>ArchivesPage</div>
-  )
-}
+    <div>
+      {archivedNotes?.map((note) => (
+        <RichTextEditor key={note._id} note={note} editNote width existingNote />
```
Comment (deepak29-git) [inline src/pages/Archives/ArchivesPage.js:11]: this is good to make a separate component

**Review by deepak29-git — APPROVED** [review]
liked: variable name is good,
your folder structure is also good

**Pull request merged**